[service-bus] changelog update - adding in SharedAccessSignature (#11021)
Azure/azure-sdk-for-js: sdk/servicebus/service-bus/CHANGELOG.md

## 7.0.0-preview.6 (Unreleased)

### New features:

- Support using the SharedAccessSignature from the connection string.
  ([PR 10951](https://github.com/Azure/azure-sdk-for-js/pull/10951)).
- Added a new field `amqpAnnotatedMessage` to the received message which will hold the received
  message in its raw form, complete with all parts of the message as per the [AMQP spec](https://www.amqp.org/sites/amqp.org/files/amqp.pdf).
- Added `ServiceBusAdministrationClient.ruleExists()`

### Breaking Changes

#### API changes

> - `SessionReceiver.sessionLockedUntilUtc` is readonly and not undefined

Reviewer: ```suggestion
- `SessionReceiver.sessionLockedUntilUtc` is readonly and never undefined.
```

  [PR 10625](https://github.com/Azure/azure-sdk-for-js/pull/10625)
- `ServiceBusClient.createDeadLetterReceiver()` has been absorbed into `createReceiver()`.
  To create a dead letter receiver:

  ```typescript
  // this same method will work with subscriptions as well.
  serviceBusClient.createReceiver(<queue>, {
    subQueue: "deadLetter"
  });
  ```

#### Renames

- The `ServiceBusManagementClient` has been renamed to `ServiceBusAdministrationClient`. See
  [Issue 11012](https://github.com/Azure/azure-sdk-for-js/issues/11012) for more details.

> - All clients are now prefixed with `ServiceBus`: `ServiceBusSender`, `ServiceBusReceiver`, `ServiceBusSessionReceiver`

Reviewer: ```suggestion
- All senders and receivers are now prefixed with `ServiceBus`: `ServiceBusSender`, `ServiceBusReceiver`, `ServiceBusSessionReceiver`
```

## 7.0.0-preview.6 (Unreleased)

### New features:

- Support using the SharedAccessSignature from the connection string.

>   ([PR 10951](https://github.com/Azure/azure-sdk-for-js/pull/10951)).

Reviewer: ```suggestion
  ([PR 10951](https://github.com/Azure/azure-sdk-for-js/pull/10951)).
  - Add a new field `amqpAnnotatedMessage` to the received message which will hold the received
  message in its raw form, complete with all parts of the message as per the [AMQP spec](https://www.amqp.org/sites/amqp.org/files/amqp.pdf).
```

## 7.0.0-preview.6 (Unreleased)

### New features:

- Support using the SharedAccessSignature from the connection string.
  ([PR 10951](https://github.com/Azure/azure-sdk-for-js/pull/10951)).

>   - Add a new field `amqpAnnotatedMessage` to the received message which will hold the received

Reviewer: ```suggestion
- Add a new field `amqpAnnotatedMessage` to the received message which will hold the received
```
